lib/hyacinth/digital_object/type_def/json_serializable_array.rb

```diff
@@ -3,7 +3,19 @@
 module Hyacinth
   module DigitalObject
     module TypeDef
-      class JsonSerializableArray < Hyacinth::DigitalObject::TypeDef::JsonSerializableBase
+      class JsonSerializableArray < Hyacinth::DigitalObject::TypeDef::JsonSerializableCollection
+        def from_serialized_form_impl(json_var)
+          return nil if json_var.nil?
```

Comment:
```suggestion
          return nil if json_var.nil?
          raise ArgumentError, "Expected array, but got: #{json_var.class}" unless json_var.is_a?(Array)
```

Reply:
I think this is a good change but there are similar oversights in one of the other classes so I'm going to push a separate commit.

Reply:
👍